guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,47 @@
+We’d like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
+dam Lazik,
+Adam Růžička,
+Aneta Šteflová Petrová,
+Archana Kumari,
+Beat Gaetzi,
+Bernhard Suttner,
+Chris Roberts,
+Cole Higgins,
+Dirk Götz,
+Eric Helms,
+Evgeni Golov,
+Ewoud Kohl van Wijngaarden,
+Girija Soni,
+Griffin-Sullivan,
+Jan Löser,
+Jeremy Lenz,
+Joniel Pasqualetto,
+Laurent Bigonville,
+Leos Stejskal,
+Marek Hulán,
+Maria Agaphontzev,
+Markus Reisner,
+Martin Alfke,
+Maximilian Kolb,
+Nadja Heitmann,
+Nofar Alfassi,
+Oleh Fedorenko,
+Partha Aji,
+Patrick Creech,
+Quinn James,
+RhinoX,
+Richard Stempfl,
+Romuald Conty,
+Samir Jha,
+Sayan Das,
+Shimon Shtein,
+William Bradford Clark,
+akumari,
+dosas,
+girijaasoni,
```

Comment:
I think we need to map these identities because they're already listed. Short term, manually
```suggestion
dosas,
```